app/src/main/java/ua/dancecamp/vmedvediev/ua21summerdancecamp/repository/Repository.kt

```diff
@@ -74,9 +88,11 @@ class Repository(private val mapper: RealmEventMapper) {
     private fun prepareEventsFromDatabase(realmEventsList: RealmResults<RealmEvent>): LinkedHashSet<ListItem> {
         val eventsList = LinkedHashSet<ListItem>()
         realmEventsList.forEach {
-            val date = mapper.from(it)
-            eventsList.add(date)
-            eventsList.add(mapper.from(it))
+            val date = eventsMapper.from(it)
+            eventsList.apply {
+                add(date)
+                add(eventsMapper.from(it))
```

Comment:
Duplication. You already have local variable for it

Reply:
In the first case I am adding object of type Date and in the second one object of type Event.

Reply:
![image](https://user-images.githubusercontent.com/26876208/44788448-bf757e00-aba2-11e8-880c-a0a400561625.png)
Fixed.